Formula/dwm.rb

```diff
@@ -41,6 +44,6 @@ def caveats
   end
 
   test do
-    assert_match /#{version}/, shell_output("#{bin}/dwm -v 2>&1", 1)
+    assert_match "no locale support", shell_output("#{bin}/dwm 2>&1", 1)
```

Comment:
That sounds like an actual problem to me?

Reply:
Sorry, I'm drawing a blank on this one. The corresponding code in `dwm.c`:
```
        if (!setlocale(LC_CTYPE, "") || !XSupportsLocale())
                fputs("warning: no locale support\n", stderr);
```
The old XQuartz-linked `dwm` doesn't throw that warning when run in the same environment, so it's likely that `XSupportsLocale()` is the call that's failing in the "new regime". @iMichka , any ideas about how this can be addressed?

That aside, I'm thinking of just setting `DISPLAY` to the empty string, and let `dwm` fail with `cannot open display`, for a reliable test with stable output. @SMillerDev , your thoughts?

Reply:
I don't think either is something that would indicate that our method is working right?

Reply:
I have not much clue about the x11 internals. I migrated the packages "as-is" from the linux repo, but I never guaranteed that they would be functional. We are here in an experimental phase, please remind that :)

So it is totally possible that something is not setup correctly, and it needs to be addressed.

Reply:
I think it just means we haven't compiled the X libraries with localization support. The formulae that I've tested so far have all worked fine in terms of e.g. using the X libraries to render to a file or whatever.

Reply:
```suggestion
    assert_match "dwm: cannot open display", shell_output("DISPLAY= #{bin}/dwm 2>&1", 1)
    assert_match "dwm-#{version}", shell_output("#{bin}/dwm -v", 1)
```